packages/ses/test/test-console-error-trap.js

```diff
@@ -1,17 +1,18 @@
 import test from 'ava';
+import url from 'url';
 import { exec } from 'child_process';
-import { dirname, join } from 'path';
 
-const cwd = join(
-  dirname(new URL(import.meta.url).pathname),
-  'console-error-trap',
-);
+const cwd = url.fileURLToPath(new URL('console-error-trap/', import.meta.url));
 
 const exitAssertions = (t, expectedCode, altExpectedCode = expectedCode) => {
   return (err, stdout, stderr) => {
-    t.log({ stdout, stderr });
+    t.log({ stdout, stderr, code: err.code, expectedCode, altExpectedCode });
+    // Unix error codes are uint8.
+    // Windows error codes are uint32.
+    // eslint-disable-next-line no-bitwise
+    const code = err.code & 0xff;
```

Comment:
Interesting, but why do we care about clamping it?

Reply:
Node.js returns `-1`, and we see that as either 0xffffffff or 0xff depending on the platform. Truncating just normalizes the code so it matches one oracle regardless of platform.

Reply:
oh fantastic! Can we add a note to that effect, I can't be the only one confused by this.

Reply:
Also why is a negative code returned?